src/rpc/util.h

```diff
@@ -16,4 +23,17 @@ CPubKey HexToPubKey(const std::string& hex_in);
 CPubKey AddrToPubKey(CKeyStore* const keystore, const std::string& addr_in);
 CScript CreateMultisigRedeemscript(const int required, const std::vector<CPubKey>& pubkeys);
 
+class DescribeAddressVisitor : public boost::static_visitor<UniValue>
```

Comment:
I think you can keep the `DescribeAddressVisitor` class entirely local to `rpc/util.cpp`, and only expose a `UniValue DescribeAddress(const CTxDestination& dest)` function.

Reply:
Changed this and `DescribeWalletAddressVisitor` to helper functions which called the visitors.